labgrid/resource/ykushpowerport.py

```diff
@@ -1,12 +1,12 @@
 import attr
 
 from ..factory import target_factory
-from .common import Resource
+from .common import NetworkResource
 
 
 @target_factory.reg_resource
 @attr.s(eq=False)
-class YKUSHPowerPort(Resource):
+class YKUSHPowerPort(NetworkResource):
```

Comment:
Why are you changing this to a network resource rather than adding a `NetworkYKUSHPowerPort`?